app/main.py

```diff
@@ -1 +1,9 @@
 # write your code here
+
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3 and "cp" in command:
+        _, source_path, destination_path = command.split()
```

Comment:
don't use command.split() twice
just create a separated variable for it and call it on demand